### DISCUSSION TITLE
copy-on-select

### DISCUSSION THREAD

**Discussion by applebait** [ghostty-org/ghostty#3874] [Feature Requests, Ideas] [answered]
I think this feature should only copy to the clipboard on mouse button release, otherwise, which is what happens right now, it's creating unnecessary copies and in my case, making me to do ⌘ + c anyways… only because I'm not sure what's in the clipboard right away.

Thanks.

**Comment by Ch00k** [comment]
In my case a new clipboard entry is created for each character in a line when I drag the selection through this line.

The screenshot shows the clipboard contents through [Alfred](https://www.alfredapp.com/).

<img width="1425" alt="Screenshot 2025-01-03 at 10 05 29" src="https://github.com/user-attachments/assets/df55c44f-405d-4528-b570-e549cc31e5dd" />

**Comment by 00-kat** [comment, marked as answer]
For the record, this was a duplicate of #4800 and a PR exists at #5995.
